src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/BufferedHtmlContent.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNet.Html.Abstractions;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.AspNet.Mvc.Rendering
+{
+    /// <summary>
+    /// Enumerable object collection which knows how to write itself.
+    /// </summary>
+    public class BufferedHtmlContent : IHtmlContent
+    {
+        private const int MaxCharToStringLength = 1024;
+        // This is not List<IHtmlContent> because that would lead to boxing all strings to IHtmlContent
```

Comment:
Did not see this comment before check in. Yes. Wrap. I'll update this in the next PR.